### PR TITLE
Add abubakar

### PR DESCRIPTION
**Pull request by Abubakarsadiqq**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1268"></a>

### CODE REVIEW

File: public/data/abubakar.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+   {
+    "name":"Abubakar sadiq",
+    "type":"personal",
+    "bio": "open source enthusiast",
+    "avatar":"https://github.com/Abubakarsadiqq.png",
+    "links": [
+        {
+
+         "name":"follow me on github",
```
Comment (Panquesito7) [inline public/data/abubakar.json:9]: I recommend capitalizing the `h` and the `g`. What do you think?
```suggestion
         "name":"follow me on GitHub",
```

**Review by kabir0x23 — CHANGES_REQUESTED** [review]
Looks good 
but there are two small issues.
1. remove the `package-lock.json` file
2. rename yourname.json file to username.json
abubakar.json -> abubakarsadiqq.json

that's it

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Could you please **Revert** the changes in `package-lock.json`, just:

1. Copy the content from this [package-lock.json](https://github.com/EddieHubCommunity/LinkFree/blob/main/package-lock.json)
2. Paste in place of the code in the `package-lock.json` file of your fork!

⚠ Also, Change your filename to `Abubakarsadiqq.json`

**Comment by Abubakarsadiqq** [comment]
> Looks good but there are two small issues.
> 
> 1. remove the `package-lock.json` file
> 2. rename yourname.json file to username.json
>    abubakar.json -> abubakarsadiqq.json
> 
> that's it


Done boss

**Review by kabir0x23 — APPROVED** [review]
Looks good ✨ 
```diff
+ abubakarsadiqq.json
```

**Review by kabir0x23 — COMMENTED** [review]
Last thing you have to do.
You just removed that file from your project, instead of from your commit

image

```diff
+ 1. Just add that file back and commit only the username.json file.
+ This file -> https://github.com/EddieHubCommunity/LinkFree/blob/main/package-lock.json 
```
That's it

**Comment by kumarsonsoff3** [comment]
Hey @Abubakarsadiqq,  

Unfortunately, you have deleted the `package-lock.json` file from your fork. (They said to **Revert** the changes)

No worries, I've restored the file for you, you just need to **merge the [PR #1](https://github.com/Abubakarsadiqq/LinkFree/pull/1/files) to your fork.** Rest will take care of itself!

**Review by ahmed82 — CHANGES_REQUESTED** [review]
To avoid committing changes to other files such as 'package-lock.json'  when you're willing to stage your new profile file just do $:/> git add abubakarsadiqq.json
NOT
$:/> git add . or -A

**Comment by Abubakarsadiqq** [comment]
Thank you alll for helping it's been greate learning from you guys , yay!!

**Review by kumarsonsoff3 — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by Vyvy-vi — APPROVED** [review]
Looks good to me!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**